packages/react-instantsearch-dom/src/lib/debounce.ts

```diff
@@ -0,0 +1,39 @@
+// Debounce the function call to the trailing edge.
```

Comment:
We could now use the typing from https://github.com/algolia/instantsearch.js/commit/3df3e8102bf72e310cd1bfbe4aa3823c30d24dbe

Reply:
done https://github.com/algolia/react-instantsearch/pull/2996/commits/65bda7dea20f1cfbc2d2fd22d22e188c2ad0dd10